cudapoa/src/cudapoa_batch.cpp

```diff
@@ -294,7 +294,7 @@ bool CudapoaBatch::reserve_buf(uint32_t max_seq_length)
 {
```

Comment:
It seems I missed this in the previous PR, change the signature of `reserve_buf(...)` to 
```
bool CudapoaBatch::reserve_buf(int32_t max_seq_length)
```
To keep it consistent with the input type and to avoid any potential overflow issues from automatic conversion between `int32_t` and `uint32_t`.

Reply:
Good catch, I'll do that

Reply:
Changed it to `int32_t`